### ISSUE TITLE
Bug: AdaptiveCard.FromJson: Default value for choiceSet type is "expanded"

### ISSUE THREAD

**Issue by Benjathing** [Microsoft/AdaptiveCards#2100]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* .NET HTML
* JavaScript

# Author or host

Author sending cards to WebChat Emulator and DirectLine.

# Version of SDK

NuGet 1.1.0

# Issue

When converting a JSON card into an Object in .NET, both 
```C#
AdaptiveCard card = jToken.ToObject<AdaptiveCard>();
```
and
```C#
AdaptiveCard card = AdaptiveCard.FromJson(cardstring);
```
result in cards, where, if they contain a ChoiceSet with no style given, result in the style being "expanded"

#### JSON
![image](https://user-images.githubusercontent.com/10608747/47060355-0124bb80-d219-11e8-977e-4240826f6fab.png)
#### C#
![image](https://user-images.githubusercontent.com/10608747/47060316-df2b3900-d218-11e8-9274-a14338dceb55.png)

# Expected result

According to the schema and documentation, "compact" is the default value.
https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/card-schema#schema-input.choiceset

# Work around

Explicitly declaring a "compact" style for the choice set in the json results in a compact choice set in the card. However, the designer doesn't do this, expecting the default value to be compact

EDIT: property is 'style', not 'type'

**Comment by khouzam** [comment]
Thanks @Benjathing for reporting this.

This is an inconsistency in our .NET renderer.

**Comment by shalinijoshi19** [comment]
To be resolved as part of addressing  #2111
